kitchen.yml

```diff
@@ -99,17 +100,26 @@ suites:
     run_list:
       - recipe[nftables-test]
 
+  - name: firewalldnew
```

Comment:
I would name it something like firewalld-dbus so it's more descriptive and intuitively goes along with your changes.